arcee/api.py

```diff
@@ -116,62 +120,13 @@ def upload_hugging_face_dataset_qa_pairs(qa_set: str, hf_dataset_id: str, datase
 
     # Load dataset from HF
     dataset = load_dataset(hf_dataset_id)
-    
-    # Convert the split to pandas
-    df = dataset[dataset_split].to_pandas()
-
-    # Loop over all the rows in df and convert the messages into QA pairs
-    for i, row in df.iterrows():
-        try:            
-            qa_pair_tuple = _chat_ml_messages_to_qa_pair(row["messages"])
-            qa_pair = {"question": qa_pair_tuple[0], "answer": qa_pair_tuple[1]}
-            qa_pairs.append(qa_pair)
-        except Exception as e:
-            print(f"Error on row {i}: {e}.  Skipping row")
-            continue
-
-    batch_size = 200
-
-    print(f"Uploading {len(qa_pairs)} QA pairs in batches of {batch_size}")
-
-    # Upload in chunks of batch_size
-    for i in range(0, len(qa_pairs), batch_size):
-        chunk = qa_pairs[i:i+batch_size]
-        print(f"Uploading {batch_size} QA pairs..")
-        upload_qa_pairs(qa_set, chunk)
-        
-    print(f"Finished uploading QA pairs")
-
-
-
-def upload_hugging_face_dataset_qa_pairs(qa_set: str, hf_dataset_id: str, dataset_split: str, data_format: str) -> None:
```

Comment:
This function was duplicated. 0 diff between them, so I removed one.

The rest of the changes inside `arcee` are fixing or adding typing and auto-formatting